packages/yoroi-extension/app/stores/stateless/tokenHelpers.js

```diff
@@ -26,62 +25,21 @@ export function getTokenName(
   return '-';
 }
 
-const ASCII_ASSET_NAME_BLACKLIST =
-  new Set<void | string>(['ADA']);
-
-function hexToValidAsciiOrNothing(hexString: string): void | string {
-  const bytes = [...Buffer.from(hexString, 'hex')];
-  const isAscii = bytes.every(b => b >= 32 && b < 127);
-  return isAscii ? String.fromCharCode(...bytes) : undefined;
-}
-
-/**
- * https://github.com/cardano-foundation/CIPs/tree/master/CIP-0067
- */
-function getCip67Tag(assetNameHEX: string): {| hexName: string, tag: ?string |} {
-  const bytes = Buffer.from(assetNameHEX, 'hex');
-  // [ 0000 | 16 bits label_num | 8 bits checksum | 0000 ]
-  if (bytes.length >= 4) {
-    const [a, b, c, d] = bytes;
-    // check 4-zero-bit brackets
-    if ((a & 0b11110000) === 0 && (d & 0b00001111) === 0) {
-      /*
-       * The 3 significant bytes are encoded within 4 bytes as:
-       * (0000 xxxx) (xxxx yyyy) (yyyy zzzz) (zzzz 0000)
-       */
-      const middleByte = (byte1, byte2) => ((byte1 & 0b1111) << 4) | ((byte2 & 0b11110000) >> 4)
-      const xByte = middleByte(a, b);
-      const yByte = middleByte(b, c);
-      const zByte = middleByte(c, d);
-      if (crc8([xByte, yByte]) === zByte) {
-        return {
-          tag: String((xByte << 8) | yByte),
-          hexName: bytes.subarray(4).toString('hex'),
-        };
-      }
-    }
+function resolveNameProperties(name: ?string): { name: string, cip67Tag: ?string } {
+  if (name == null || name.length === 0 || !isHexadecimal(name)) {
+    return { name: '', cip67Tag: null };
   }
+  const { asciiName, hexName, cip67Tag } =
+    AssetNameUtils.resolveProperties(name ?? '');
```

Comment:
`?? ''` seems not necessary because previous `if` covers this case

Reply:
Ah right! A remnant from before the check got added